azavea/ism-watershed-wellness-snapshot
# Remove fullscreen code

## Overview

In #87, we attempted to come up with a solution to making the web app appear like a native iOS app. Over time, various options to customize the browser chrome or open the app in a dedicated browser have been removed from iOS/Safari. We ended up adding code that triggers fullscreen mode in Safari when the app was initially interacted with.

Through further testing, this has proved to be a problematic solution. First, while the app did initially enter fullscreen mode, it also sometimes exited and then immediately returned fullscreen mode. This
lead to glitchy interactions with the app. Secondly, in iOS v12, Safari now triggers a popup warning when interacting with apps in fullscreen mode (likely because of malicious use by some sites) that pops up after tapping on the app a few times. This popup made the app unusable.

Connects #93

### Demo

Screenshot of app on `develop` with fullscreen code. This happens after tapping the app a few times.

![Image-1](https://user-images.githubusercontent.com/1042475/62384657-f2e22a80-b520-11e9-8800-2f411a9b1027.jpg)

### Notes

See #87 for more background on this.

## Testing Instructions

- Visit the app on an iPad. Verify it is no longer in fullscreen mode, and you don't get the keyboard warning popup.
